src/components/Tree/Node.tsx

```diff
@@ -141,27 +135,33 @@ export const Node = ({
             >
                 <div
                     className={merge([
-                        ' tw-flex tw-py-1 tw-px-2 tw-rounded tw-no-underline tw-leading-6',
+                        ' tw-flex tw-py-2 tw-px-2.5 tw-no-underline tw-leading-5',
                         strong && 'tw-font-bold',
-                        value && !selected && 'hover:tw-bg-black-5',
-                        selected ? 'tw-bg-violet-60 tw-text-white' : 'tw-text-black',
+                        value && !selected && 'hover:tw-bg-black-10 hover:tw-text-box-neutral-inverse-hover',
+                        selected
+                            ? 'tw-bg-violet-60 tw-text-white hover:tw-bg-box-selected-strong-hover hover:tw-text-box-selected-strong-inverse-hover'
+                            : 'tw-text-black',
```

Comment:
I think that the ternary condition overwrites the default text styling and sets it to black when the item is not selected.